src/Insect/Environment.purs

```diff
@@ -108,8 +108,17 @@ initialEnvironment =
         [ constFunc  "abs" (Q.abs >>> pure)
         , constFunc  "acos" Q.acos
         , constFunc  "acosh" Q.acosh
-        , constFunc  "acos" Q.acos
-        , constFunc  "acosh" Q.acosh
+        , constFunc  "acot" Q.acot
+        , constFunc  "arccotangent" Q.acot
```

Comment:
I used "arc" for the full names of the inverse functions, but we could also use "a" instead (e.g. "acotangent"). What do you think @sharkdp?

Reply:
I like the `arc` prefix. It's also how those functions (e.g. arctangent, arcsine) are called on Wikipedia.